test/controllers/register/beneficiaries/AddABeneficiaryControllerSpec.scala

```diff
@@ -223,6 +230,12 @@ class AddABeneficiaryControllerSpec extends SpecBase {
 
       "return OK and the correct view for a GET" in {
 
+        val settlorsAnswers: ReadOnlyUserAnswers = ReadOnlyUserAnswers(
```

Comment:
If you change getSettlorsAnswers to return the common base trait (ReadableUserAnswers), then you don't need to have this double level of creation to make a ReadOnlyUserAnswers. I'm not sure how that happened along the way.